frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/strings/strings.ts

```diff
@@ -4,7 +4,7 @@ export const CREATE_VM = 'Create Virtual Machine';
 export const CREATE_VM_TEMPLATE = `${CREATE_VM} Template`;
 export const IMPORT = `Import`;
 export const IMPORT_VM = 'Import Virtual Machine';
-export const REVIEW_AND_CREATE = 'Review and create';
+export const REVIEW_AND_CONFIRM = 'Review and confirm';
 export const REVIEW_AND_IMPORT = 'Review and import';
```

Comment:
In that case, we can use the same label for create and import

Reply:
thanks :+1:

Reply:
fixed :cake: